src/libmongoc/tests/json-test-monitoring.c

```diff
@@ -442,6 +442,25 @@ _apm_match_error_context (const bson_t *actual, const bson_t *expectations)
    bson_free (actual_str);
    bson_free (expectations_str);
 }
+
+bool skip_cse_list_collections(const bson_t *doc) {
+   // see CDRIVER-3856: Sharing a MongoClient for metadata lookup can lead to
+   // deadlock in drivers using automatic encryption
```

Comment:
Use `/* */` comments for c89.

Can you note in this comment that the C driver explicitly does not use a separate `mongoc_client_t` for listCollections / find on the key vault, which is why we skip checks for them?

Reply:
Done